cantera/build_py.bat

```diff
@@ -2,20 +2,25 @@ echo ****************************
 echo PYTHON %PYTHON% BUILD STARTED
 echo ****************************
 
-COPY cantera.conf cantera.conf.bak
-DEL /F cantera.conf
-FINDSTR /V "python_package" cantera.conf.bak > cantera.conf
-DEL /F cantera.conf.bak
+:: Remove old Python build files, if they are present
+IF EXIST "build/python" RD /S /Q "build/python"
+IF EXIST "build/temp-py" RD /S /Q "build/temp-py"
+IF EXIST "%ESC_PREFIX%/bin/ck2yaml" DEL /F "%ESC_PREFIX%/bin/ck2yaml"
+IF EXIST "%ESC_PREFIX%/bin/cti2yaml" DEL /F "%ESC_PREFIX%/bin/cti2yaml"
+IF EXIST "%ESC_PREFIX%/bin/ctml2yaml" DEL /F "%ESC_PREFIX%/bin/ctml2yaml"
+IF EXIST "%ESC_PREFIX%/bin/yaml2ck" DEL /F "%ESC_PREFIX%/bin/yaml2ck"
```

Comment:
I don't think `ESC_PREFIX` is defined in this file?  Anyway, since this is a Windows command, you can just use the regular `PREFIX`. I also don't think this needs to be conditional, the `/F` flag should avoid failing if it doesn't exist, IIRC

Reply:
Thanks! I fixed the `ESC_PREFIX` (although I ultimately believe that the files will be overwritten regardless), but left the `IF` in as I'm not sure that `DEL /F` won't produce a fail (it's definitely a forced delete, but I'd rather not wait for an hour-long build to find out). I think further tweaks can be done in subsequent PR's ... main thing is that the `conda` packages work again.